lib/install/config/webpack/paths.yml

```diff
@@ -0,0 +1,7 @@
+# Restart webpack-watcher or webpack-dev-server if you make changes here
+paths:
+  src_path: app/javascript
+  config_path: config/webpack
+  node_modules_path: node_modules
+  dist_dir: packs
+  dist_path: public/packs
```

Comment:
Also, given that these are all keyed under `paths`, we could drop the `_path` suffix.
```yaml
paths:
  config: ...
  src: ...
  dist: ...
```